CODE/Sources/PennyBOT/CommandsManager.swift

```diff
@@ -10,7 +10,7 @@ struct CommandsManager {
 
     private func makeCommands() -> [Payloads.ApplicationCommandCreate] {
```

Comment:
In main-branch commits i've done some clean ups around how we handle Application/Slash commands.
Penny now uses enums to cleanly register commands to Discord, and also cleanly decode them in `InteractionHandler` when receiving from Discord.